app/src/main/java/com/kickstarter/services/DiscoveryParams.java

```diff
@@ -415,7 +415,7 @@
    * POTD comes back.
    */
   public boolean shouldIncludePotd() {
-    return isAllProjects() && page() != null && page() == 1 && (sort() == null || sort() == Sort.HOME);
+    return isAllProjects() && page() != null && page() == 1 && (sort() == null && (term() == null || term().isEmpty()) || sort() == Sort.HOME);
```

Comment:
I think the parentheses got a little messed up here. if you expand it a bit you will see it's written as:

```swift
return isAllProjects() 
  && page() != null 
  && page() == 1 
  && (
    sort() == null 
    && (term() == null || term().isEmpty()) 
    || sort() == Sort.HOME
);
```

since `||` has a lower precedence than `&&` we will get `true` for this expression if `sort() == Sort.HOME` and will completely discard the `sort() == null` and `term()` portion of the condition.

what you really want is more like:

```java
return isAllProjects()
  && page() != null 
  && page() == 1 
  && (sort() == null || sort() == Sort.HOME)
  && (term() == null || term().isEmpty());
```